### PR TITLE
Create dev build for 1.3.2

### PR DESCRIPTION
**Pull request by aws-pangestu**
Create dev build for 1.3.2

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)

* [x] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/conda_build_config.yaml [hunk 1]
```diff
@@ -1,2 +1,2 @@
-channel_targets:
-  - conda-forge sagemaker-code-editor_dev
```
Comment (navinsoni) [inline recipe/conda_build_config.yaml:2]: here we are removing dev, is it ok.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,40 +1,41 @@
...
   skip: true  # [not linux]
 
 requirements:
   build:
+    - compilers
     # dependencies needed for open source microsoft/vscode - https://github.com/microsoft/vscode/wiki/How-to-Contribute 
-    - nodejs >=20.9
+    # updating nodejs to >=20.0 as it's requirement for vscode - https://github.com/microsoft/vscode/wiki/How-to-Contribute
+    # vscode has dependency on @azure/msal-node@2.5 which expects node version 20
+    - nodejs >=20.0,<21
```
Comment (navinsoni) [inline recipe/meta.yaml:23]: why are we limiting this to <21

Comment (aws-pangestu) [reply recipe/meta.yaml:23]: this was just merged in from main

File: recipe/conda_build_config.yaml (hunk 1 above)
Comment (aws-pangestu) [reply recipe/conda_build_config.yaml:2]: good catch, i'll fix this

File: recipe/meta.yaml (hunk 2 above)
Comment (aws-asolidu) [reply recipe/meta.yaml:23]: @navinsoni  See the comment. This is needed for dependencies in vscode.

**Review by aws-asolidu — APPROVED** [review]

**Pull request merged**